### DISCUSSION TITLE
Use with Echo?

### DISCUSSION THREAD

**Discussion by edalzell** [qruto/laravel-wave#16] [Q&A] [answered]
Hey there, nice package.

I currently have my chat app wired up with Beyond Code's Websockets, but I'm very interested in having it work with SSEs.

If I use the wave server side code, can I use echo/livewire the same on the front end?

**Comment by edalzell** [comment]
Ok I see everything almost working, but I don't know what the "event" is? In Livewire you choose `private`, `public` and `App.Models.User.{user}` or something.

What do I need to put to make it work in Echo?

**Comment by edalzell** [comment, marked as answer]
Haha the answer is everything works fine the way it was, and I'm an idiot

**Comment by slavarazum** [comment]
@edalzell Hey 👋
Let me know if you will have any issues and enjoy of using Wave.
